data/secscan_model/test/test_secscan_interface.py

```diff
@@ -1,32 +1,73 @@
+import pytest
```

Comment:
Can you define another test for the translation of V4 results into the API-expected JSON? It should take in a V4 vulnreport document and compare it to a golden-defined output doc

Reply:
Done [here](https://github.com/quay/quay/pull/259/files#diff-d1556d15ee5435b7ee2ca49b3a797803R231).